recipes/mold/all/conanfile.py

```diff
@@ -1,5 +1,8 @@
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.scm import Version
+from conan.tools import files
+from conan.errors import ConanInvalidConfiguration
+from conans import AutoToolsBuildEnvironment
 import os
 
 required_conan_version = ">=1.45.0"
```

Comment:
```suggestion
required_conan_version = ">=1.47.0"
```
https://github.com/conan-io/conan-center-index/blob/master/docs/v2_linter.md#import-tools-from-conan